templates/components/post_comment.html.twig

```diff
@@ -207,7 +212,9 @@
             comment: comment,
             level: level,
             showNested: true,
-            view: app.request.cookies.get(constant('App\\Controller\\User\\ThemeSettingsController::POST_COMMENTS_VIEW'), 'tree')
+            view: VIEW_STYLE
```

Comment:
i'm mostly certain this isn't used. I see no visible way to change the view style when viewing microblogs, and I tried writing the cookie out manually as value tree/chat/classic and none of them changed the view of 1 single indent for replies. possibly this can all be removed

Reply:
going to leave it alone just in case. I do believe it can't be changed like entry comment can, but there are different styles depending on if it was a threaded entry that somehow ended up in the post tab or an actual microblog. might need to look at it more later to see, it might depend more on the source/type than user setting, but just won't touch it for now

Reply:
Oh, I was indeed wrong about this, a way to change the view style does appear on the single post view. It became more apparent with a friendica thread that ended up in the microblog section